### PR TITLE
Bugfix/show flag status in partners list

### PR DESCRIPTION
**Pull request by kenzo23312**

### CODE REVIEW

File: frontend/src/components/partners/partnerProfileNameCell.js [hunk 1]
```diff
@@ -15,6 +16,12 @@ const styleSheet = (theme) => {
       alignItems: 'center',
       cursor: 'pointer',
     },
+    iconUnverified: {
+      fill: theme.palette.error[500],
+      width: 15,
```
Comment (marcindo) [inline frontend/src/components/partners/partnerProfileNameCell.js]: will be better now to extract common part from all classes: width, height, margin to separate class

File: frontend/src/components/partners/partnerProfileNameCell.js [hunk 2]
```diff
@@ -44,30 +51,31 @@ const styleSheet = (theme) => {
 
 /* eslint-disable jsx-a11y/interactive-supports-focus */
 const PartnerProfileNameCell = (props) => {
```
Comment (marcindo) [inline frontend/src/components/partners/partnerProfileNameCell.js]: would be better to have separate component that displays partner name along with badge and flags, use it here wrapped in TableCell

Comment (kenzo23312) [reply frontend/src/components/partners/partnerProfileNameCell.js]: done

File: frontend/src/components/partners/partnerProfileNameCell.js (hunk 1 above)
Comment (kenzo23312) [reply frontend/src/components/partners/partnerProfileNameCell.js]: done

**Pull request merged**